kore/src/Kore/Unification/UnificationProcedure.hs

```diff
@@ -0,0 +1,36 @@
+{- |
+Copyright   : (c) Runtime Verification, 2019
```

Comment:
```suggestion
Copyright   : (c) Runtime Verification, 2021
```

Reply:
I think this file was deleted from master but not from my branch and that's why it appears like I added it. Thanks for noticing